install/0000_00_cluster-version-operator_01_admingate_configmap.yaml

```diff
@@ -1,9 +1,14 @@
 apiVersion: v1
 kind: ConfigMap
+data:
+  ack-4.11-kube-1.24-api-removals-in-4.12: |
+      Kubernetes 1.24 and therefore OpenShift 4.12 remove several APIs which require admin consideration. Please see
+      the knowledge article https://access.redhat.com/articles/6955381 for details and instructions.
 metadata:
   name: admin-gates
   namespace: openshift-config-managed
   annotations:
     kubernetes.io/description: Define update gates for the administrator to acknowledge in the admin-acks ConfigMap in the openshift-config namespace.
     include.release.openshift.io/ibm-cloud-managed: "true"
     include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
```

Comment:
can we drop this?  It's dead code as I described in #685.

Reply:
Fixed in https://github.com/openshift/cluster-version-operator/commit/2d66c169658a72e1c54272646d352fc10f378b6f